src/hooks/useDateCheck.js

```diff
@@ -1,32 +1,31 @@
 import { useEffect } from "react";
 import { format, parseISO, isBefore } from "date-fns";
-import { useAppState } from "../AppContext";
+import { useAppState, useAppReducer } from "../AppContext";
 const { remote } = require("electron");
 
 export default function useDateCheck() {
   const { date } = useAppState();
+  const dispatch = useAppReducer();
   const storedDate = parseISO(`${date.year}-${date.month}-${date.day}`);
 
   useEffect(() => {
-    let nd = new Date();
-
     const interval = setInterval(() => {
+      let nd = new Date();
       let currentDate = parseISO(
         `${format(nd, "y")}-${format(nd, "MM")}-${format(nd, "dd")}`
       );
+      console.log(remote.getCurrentWindow());
 
       if (isBefore(storedDate, currentDate)) {
         if (remote.getCurrentWindow().showResetNotification) {
-          // Disabling this line so that it doesn't yell at us for not using
-          // the notification variable
-          // eslint-disable-next-line
-          let resetNotification = new Notification("todometer reset time!", {
+          new Notification("todometer reset time!", {
             body: "It's a new day! Your todos are being reset."
           });
         }
+        dispatch({ type: "RESET_ALL" });
         window.location.reload();
       }
     }, 1000);
```

Comment:
Try to set the interval millisecond to the next day and the current date difference.
`differenceInMilliseconds(endOfDay(nd), nd)`

Reply:
Did that before, it ended up bugging out when a computer went to sleep. Thanks though!